### DISCUSSION TITLE
Based on Parent value condition hide or show the HasManyRepeater input components

### DISCUSSION THREAD

**Discussion by webtamizhan** [filamentphp/filament#2693] [General]
Does HasManyRepeater Component support reactive update of the parent component?

Say for an example, i have the following checkbox component. Whenever the checkbox selected i need to show or hide the component inside the HasManyRepeater.

` Forms\Components\Checkbox::make('status')`
`->default(true)`
`->reactive(),`

HasManyRepeater snippet:

`Forms\Components\HasManyRepeater::make('items')`
    `->relationship('items')`
    `->reactive()`
    `->schema([ `
      `       Forms\Components\TextInput::make('price')`
         `    ->label('Price')`
          `->numeric()`
          `->required()`
          `->columnSpan([`
            `  'md' => 3,`
          `]),`
      `Forms\Components\TextInput::make('discount_price') `
          `->hidden(fn(Closure $get) => !$get('status'))`
          `->reactive()`
         ` ->required()->columnSpan([`
             ` 'md' => 4,`
         ` ]), `
    `])`
    `->createItemButtonLabel('Add Pricing')`
    `->defaultItems(1)`
    `->disableLabel()`
    `->columns([`
     ` 'md' => 12,`
    `])`
    `->required(),`

**Comment by danharrin** [comment]
You can use `../` to access values outside the repeater, so `$get('../../status')`
